src/adapter/src/flags.rs

```diff
@@ -152,6 +151,7 @@ pub fn storage_config(config: &SystemVars) -> StorageParameters {
         enable_dependency_read_hold_asserts: config.enable_dependency_read_hold_asserts(),
         user_storage_managed_collections_batch_duration: config
             .user_storage_managed_collections_batch_duration(),
+        dyncfg_updates: Some(config.dyncfg_updates()),
```

Comment:
this makes the copy in PersistParameters above redundant, we should probably remove that one

```
        persist: PersistParameters {
            config_updates: config.dyncfg_updates(),
        },
```

Reply:
Oh that's awesome!